metadata-io/src/main/java/com/linkedin/metadata/entity/ebean/EbeanAspectDao.java

```diff
@@ -206,31 +186,25 @@ private void saveEbeanAspect(
   }
 
   @Override
-  public Map<String, Map<String, EntityAspect>> getLatestAspects(
-      @Nonnull Map<String, Set<String>> urnAspects) {
+  public Map<String, Map<String, EntityAspect>> getLatestAspects(@Nonnull Map<String, Set<String>> urnAspects) {
     validateConnection();
 
-    List<EbeanAspectV2.PrimaryKey> keys =
-        urnAspects.entrySet().stream()
```

Comment:
There are a lot of reformatting changes that got put into this PR that make it hard to see what has actually been changed

Reply:
That's all from this PR: https://github.com/datahub-project/datahub/pull/9373
Do you know what was run to do the formatting for that PR so that I can run it on my changes?

Reply:
I believe it's through Spotless which seems to be run through `./gradlew spotlessApply`, might need to be run at a per module basis I haven't used it yet personally so haven't worked through the kinks myself yet. It might also work itself out when updating the branch which we can try.

Reply:
Yeah merge did not fix it, it's showing that the lines got specifically changed from the new format to old format in this PR 😕

Reply:
The build output gives the command: `./gradlew :metadata-service:services:spotlessApply` should fix it